ruleset/rules/0690-gcp_rules.xml

```diff
@@ -283,6 +283,245 @@ ID: 65000 - 65499
+        <description>GCP logging bucket deleted</description>
+        <mitre>
+            <id>T1562</id>
+        </mitre>
+    </rule>
+
+    <!-- defense evasion gcp_logging_sink deletion -->
+    <rule id="65057" level="3">
+        <if_sid>65000</if_sid>
+        <field name="gcp.protoPayload.methodName" type="pcre2">^google\.logging\.v\d*\.ConfigServiceV\*\.DeleteSink$</field>
+        <field name="gcp.protoPayload.authorizationInfo" type="pcre2">"granted":\strue</field>
+        <description>GCP logging sink deleted</description>
+        <mitre>
+            <id>T1562</id>
+        </mitre>
+    </rule>
+
+    <!-- defense evasion gcp_pub_sub_subscription deletion -->
+    <rule id="65058" level="3">
+        <if_sid>65000</if_sid>
+        <field name="gcp.protoPayload.methodName" type="pcre2">^google\.pubsub\.v\d*\.Subscriber\.DeleteSubscription$</field>
+        <field name="gcp.protoPayload.authorizationInfo" type="pcre2">"granted":\strue</field>
+        <description>GCP pub/sub subscription deleted</description>
+        <mitre>
+            <id>T1562</id>
+        </mitre>
+    </rule>
+
+    <!-- defense evasion gcp_pub_sub_topic deletion -->
+    <rule id="65059" level="3">
+        <if_sid>65000</if_sid>
+        <field name="gcp.protoPayload.methodName" type="pcre2">^google\.pubsub\.v\d*\.Publisher\.DeleteTopic$</field>
+        <field name="gcp.protoPayload.authorizationInfo" type="pcre2">"granted":\strue</field>
+        <description>GCP pub/sub topic deleted</description>
+        <mitre>
+            <id>T1562</id>
+        </mitre>
+    </rule>
+
+    <!-- defense evasion gcp_storage_bucket_configuration modified -->
+    <rule id="65060" level="3">
+        <if_sid>65000</if_sid>
+        <field name="gcp.protoPayload.methodName" type="pcre2">^storage\.buckets\.update$</field>
+        <field name="gcp.protoPayload.authorizationInfo" type="pcre2">"granted":\strue</field>
+        <description>GCP storage bucket configuration modified</description>
+    </rule>
+
+    <!-- defense evasion gcp_storage_bucket_permissions modified -->
+    <rule id="65061" level="3">
+        <if_sid>65000</if_sid>
+        <field name="gcp.protoPayload.methodName" type="pcre2">^sstorage\.setIamPermissions$</field>
+        <field name="gcp.protoPayload.authorizationInfo" type="pcre2">"granted":\strue</field>
+        <description>GCP storage bucket permissions modified</description>
+        <mitre>
+            <id>T1222</id>
+        </mitre>
+    </rule>
+
+    <!-- exfiltration gcp_logging_sink modification -->
+    <rule id="65062" level="3">
+        <if_sid>65000</if_sid>
+        <field name="gcp.protoPayload.methodName" type="pcre2">^sgoogle\.logging\.v\d*\.ConfigServiceV\d*\.UpdateSink$</field>
+        <field name="gcp.protoPayload.authorizationInfo" type="pcre2">"granted":\strue</field>
+        <description>GCP logging sink modified</description>
+        <mitre>
+            <id>T1537</id>
```

Comment:
I think it's more related to T1562 than T1537. There is a similar one some rules before this one

Reply:
Could be both, but I'm going to trust Elastic on this one.